README.md

```diff
@@ -3,13 +3,23 @@
 Every player in the competition creates his version of of the ai-snake class.
 The players need to implement the 'change_direction' function so their snake will be 'smart' enough to survive and win.
 
+Some information about the game:
+
+Game settings: 
+1. We now need to specify the number of cells we want in the board. No need to mess with height, width or cell size.
+
+2. You can press + or - keys to adjust the delay between frames.
+
+
+Logic:
+1. When a snake hits itself is cuts itself in the hitting point.
```

Comment:
Nice.
We might want to change the scoring system if so - because then if someone had a really long snake and it is cut in the last second, he will lose.
Maybe change it to be the average of the longest the snake has been and the final length of the snake?
Just an idea, maybe needs some more thinking.